modules/ClinikalAPI/Module.php

```diff
@@ -113,20 +115,43 @@ public function getServiceConfig()
                 },
                 */
                 ManageTemplatesLettersTable::class =>  function(ContainerInterface $container) {
-
                     $dbAdapter = $container->get(\Laminas\Db\Adapter\Adapter::class);
                     $resultSetPrototype = new ResultSet();
                     $resultSetPrototype->setArrayObjectPrototype(new ManageTemplatesLetters());
                     $tableGateway = new ZendTableGateway('manage_templates_letters', $dbAdapter, null, $resultSetPrototype);
                     $table = new ManageTemplatesLettersTable($tableGateway);
                     return $table;
                 },
+                'FormDiagnosisAndRecommendationsQuestionnaireMapTable' =>  function(ContainerInterface $container) {
+                    $dbAdapter = $container->get(\Laminas\Db\Adapter\Adapter::class);
+                    $resultSetPrototype = new ResultSet();
+                    $resultSetPrototype->setArrayObjectPrototype(new QuestionnaireMap());
+                    $tableGateway = new ZendTableGateway('form_diagnosis_and_recommendations_questionnaire', $dbAdapter, null, $resultSetPrototype);
+                    $table = new QuestionnaireMapTable($tableGateway);
+                    return $table;
+                },
+
+                'FormMedicalAdmissionQuestionnaireMapTable'  =>  function(ContainerInterface $container) {
```

Comment:
the form_medical_admission_questionnaire is specific questannaire for emergency, the create table statement is found in the emergency module, need to move this function to EmergencyMedicine/Module.php

Reply:
resolved